packages/orbit-components/src/BadgeList/BadgeListItem/index.tsx

```diff
@@ -52,8 +52,8 @@ export const StyledVerticalBadge = styled.div<{ $type?: Props["type"] }>`
     justify-content: center;
     margin-${right}: ${theme.orbit.spaceXSmall};
     flex-shrink: 0;
-    height: ${theme.orbit.heightIconMedium};
-    width: ${theme.orbit.widthIconMedium};
+    height: ${theme.orbit.heightIconLarge};
```

Comment:
so if I got it right, only the text should be changed and the icon should be always large (24px)? right?

Reply:
I am not sure, because of what I reported that icon sizes in Figma (or the icon placeholders used there) are too small to be perceived. I'd wait on designers' feedback on this one